Automattic/jetpack
# Social Menus & Icons: Add Discord Support

Resolves #11093 

I'm not quite sure what the deal is with social icons. In testing these changes, this seems to work for social menus, but my understanding is that Automattic/jetpack#37549 would need to be addressed before it's possible to add a Discord social icon. I also don't really understand the CSS bit of this file, it doesn't seem necessary to add? Is that just for social icons too? Please correct me if that isn't the case! 

#### Changes proposed in this Pull Request:


Add Discord support to the social menu. `discord.com` is another URL, but the main use of Discord on a social menu is to provide an invite to a server, and that just redirects. The main two URLs for that are `discord.gg` and `discordapp.com`

![screenshot_20190203-093216](https://user-images.githubusercontent.com/43215253/52175117-a8146f80-2796-11e9-864e-4c31a02e3869.jpg)

#### Testing instructions:





Go to the Customizer with a theme that supports social menus and add a social menu using one of the two Discord links above, then verify the icon displays as expected. 

![fgsddfgssfgdgfsd](https://user-images.githubusercontent.com/43215253/52294950-7bb54a80-2972-11e9-8e4e-134b2ee8bc39.png)



#### Proposed changelog entry for your changes:


Social Menus: Add Discord support
